client/src/pages/Prompt.js

```diff
@@ -6,51 +6,120 @@ import {
   getGameLoop,
   gameStage,
   gameStageURL,
-  checkCorrectGame } from "../util";
-import { queryBackend, queryBackendOnErr, startPinging, instantiateGameUpdater, leaveGame } from "../net";
+  checkCorrectGame,
+} from "../util";
+import {
+  queryBackend,
+  queryBackendOnErr,
+  startPinging,
+  instantiateGameUpdater,
+  leaveGame,
+} from "../net";
 import { NavLink } from "react-router-dom";
-import { Button, Card } from "react-bootstrap";
 import Play from "../components/Play";
+import Timer from "../components/Timer";
 
 const Prompt = () => {
-  window.addEventListener('beforeunload', e => {
+  window.addEventListener("beforeunload", (e) => {
     leaveGame();
   });
 
   const navigate = useNavigate();
-  const code = useParams().id;
+  const gameID = useParams().id;
+  const username = loadLocalData().userName;
+  const privateKey = loadLocalData().privateKey;
+  const roundLength = loadLocalData().roundLength;
   const [gameData, setGameData] = useState({});
+  const startTime = gameData.startTime;
+  const [successfulSubmission, setSuccessfulSubmission] = useState(null);
+
+  const players = gameData["players"];
+  const host = gameData["host"];
 
   useEffect(() => {
-    checkCorrectGame(code, navigate);
-    instantiateGameUpdater(gameStage.PROMPT, setGameData, navigate); 
-  }, [code, setGameData]);
+    checkCorrectGame(gameID, navigate);
+    instantiateGameUpdater(gameStage.PROMPT, setGameData, navigate);
+  }, [gameID, setGameData]);
 
   const submitHandler = (response) => {
-    console.log(response)
+    queryBackend(
+      "submitPlayerResponse",
+      {
+        id: gameID,
+        name: username,
+        privateKey: privateKey,
+        playerResponse: response,
+      },
+      () => {
+        setSuccessfulSubmission(true);
+      }
+    );
   };
 
   const createBody = () => {
-    const me = loadLocalData().userName;
     if (gameData["players"] === undefined) {
       return "";
-    } else if (gameData["host"] === me) {
-      return (<div>
-        <h1 className="prompt">Waiting for players to submit responses.</h1>
-      </div>);
+    } else if (username === gameData["judge"]) {
+      return (
+        <div className="home-button-container">
+          <h1 className="prompt">
+            You are the judge. <br />
+          </h1>
+
+          <div className="prompt-timer">
+            <Timer roundLength={roundLength} startTime={startTime} />
+          </div>
+
+          {Object.values(players)
+            .sort()
+            .map((player, index) => {
+              return player.name === host ? (
```

Comment:
Why are you ignoring the host here?

Reply:
This indicates who has and hasn't submitted yet. The host can't submit so they're excluded from the list